apps/web/src/components/borrow/table/TableInput.tsx

```diff
@@ -10,17 +10,32 @@ import {
 } from '@bloxifi/ui'
 import React, { ForwardedRef, forwardRef } from 'react'
 import { useTranslation } from 'react-i18next'
+import { TokenList } from 'packages/core/src'
 
 import { ReservesData } from '@/containers/WalletContainer'
 
+type TableInputData = {
```

Comment:
We can inline here
`reserveData: TableInputData` to
`reserveData: Pick<ReserveData,'balance'|'symbol'>`
much better when we refactor ReserveData (typescript will show us the way when that happens.)